src/screens/PostScreen.tsx

```diff
@@ -36,36 +57,27 @@ export default function PostScreen() {
   return (
     <View style={styles.container}>
       <View style={styles.post}>
-        <View style={styles.personHeader}>
+        <Pressable style={styles.personHeader} onPress={fetchData}>
           <View style={styles.nameProfile}>
             <ProfilePlaceholder />
-            <Text style={styles.username}>rbeggs</Text>
+            <Text style={styles.username}>{postData['username']}</Text>
           </View>
           <View style={styles.dateDiv}>
-            <Text style={styles.postDate}>September 19</Text>
+            <Text style={styles.postDate}>{postData['created_at']}</Text>
           </View>
-        </View>
-        <Text style={styles.caption}>
-          In response to the growing homelessness crisis in San Francisco, a
-          local nonprofit organization, Code Tenderloin, has launched a
-          comprehensive initiative aimed at providing long-term solutions for
-          individuals experiencing homelessness. The organization, founded in
-          2015, is dedicated to addressing both immediate needs and underlying
-          causes of homelessness through a combination of shelter services, job
-          training programs, and mental health support. read more online:
-          <Text style={styles.link}>https://www.codetenderloin.org/</Text>
-        </Text>
+        </Pressable>
+        <Text style={styles.caption}>{postData['description']}</Text>
         <Image
           style={styles.postImage}
           source={{
-            uri: 'https://cdn.britannica.com/51/178051-050-3B786A55/San-Francisco.jpg',
+            uri: postData['image_url'],
```

Comment:
I typically use the `postData.image_url` dot syntax for accessing object attributes. Use whatever is consistent with the rest of your project.